pkg/cmd/pipeline/describe_test.go

```diff
@@ -0,0 +1,229 @@
+package pipeline
```

Comment:
Ah forgot that earlier : you need to have those header in all file (so even on tests)

Reply:
Sure.